api/lib/source/cosmosV0-source.js

```diff
@@ -510,10 +521,15 @@ class CosmosV0API extends RESTDataSource {
     )
   }
 
-  async getOverview(delegatorAddress, validatorsDictionary, fiatCurrency) {
+  async getOverview(
```

Comment:
we can remove this, right?

Reply:
it seems it is still being used, that is why I had to change it

Reply:
Where is it used?

Reply:
DelegationModal

Reply:
Let'S remove there

Reply:
It is also being used in `transaction-manager.js` to get `accountInformation`. Should I remove it from there too?

Reply:
yep

Reply:
Done